### ISSUE TITLE
Project does not compile anymore after updating to 5.10.0

### ISSUE THREAD

**Issue by pepejeria** [facebook/facebook-ios-sdk#1105]
## Environment

Describe your dev environment here, giving as many details as possible. If you have them, make sure to include:

- Xcode Version: `11.2`
- Swift Version: `5` (if issue is Swift related)
- Installation Platform & Version: Used `Cocoapods 1.8.4` to install version `5.10.0`

## Expected Results

Project should compile without any problems

## Actual Results

Project does not compile

## Steps to Reproduce

Using Cocoapods, do a pod update to get the latest version 5.10.0 (was 5.8.0).

Note, this was working fine with version 5.8.0

## Error
```
/Users/jose/Development/SocialSteps/Pods/FBSDKLoginKit/FBSDKLoginKit/FBSDKLoginKit/Internal/FBSDKLoginManager+Internal.h:21:9: 'FBSDKCoreKit+Internal.h' file not found
```

## Pod

```
    pod 'FBSDKCoreKit'
    pod 'FBSDKLoginKit'
```

**Comment by CoCoE1203** [comment]
'FBSDKCoreKit is not found' to

SDK version 5.10.0

Help Please

**Comment by thiagoos** [comment]
Same problem here, I was search for answer for it.

**Comment by Guerrix** [comment]
Any update on this? I having the exact same issue. Thanks

**Comment by pcjbird** [comment]
Any update?

**Comment by TengkuFathullah** [comment]
I am having the same issue, anyone discovered a quick fix for now?

**Comment by tgensol** [comment]
Check this issue : https://github.com/facebook/facebook-ios-sdk/issues/1102

**Comment by Guerrix** [comment]
Thanks @tgensol 

Base on what I saw a valid workaround(not a fix tho) is to add ` :modular_headers => true`:
```
pod 'FBSDKCoreKit', :modular_headers => true
```

It solved the issue for me.

**Comment by joesus** [comment]
Patch release 5.10.1 is live. Please open any new issues related to that release specifically. Thanks!

**Comment by pepejeria** [comment]
@joesus this is still not fixed. I am still getting the following error:
`/Users/jose/Development/SocialSteps/Pods/FBSDKLoginKit/FBSDKLoginKit/FBSDKLoginKit/Internal/FBSDKLoginManager+Internal.h:21:9: 'FBSDKCoreKit+Internal.h' file not found`

Installed version 5.10.1 using Cocoapods. Deleted derived data + clean + build. Using Xcode Version 11.2.1 (11B500)

**Comment by joesus** [comment]
@pepejeria I've just tested this again in a fresh project running:

`pod init`

Updating the podfile:
```
# Uncomment the next line to define a global platform for your project
# platform :ios, '9.0'

target 'TestPods' do
  # Comment the next line if you don't want to use dynamic frameworks
  use_frameworks!

  # Pods for TestPods
  pod 'FBSDKCoreKit'
  pod 'FBSDKLoginKit'
  pod 'FBSDKShareKit'

end
```

and importing the kits into ViewController.swift

Everything builds as expected. Could you try also running `pod deintegrate` and reinstalling/updating the pods?

**Comment by pepejeria** [comment]
@joesus I ran `pod reintegrate` + `pod install` with no luck. Still get the `#import "FBSDKCoreKit+Internal.h" file not found`

**Comment by joesus** [comment]
It sounds like might be specific to your environment. You're more than welcome to submit a PR that accounts for the specific use case but if I cannot reproduce the issue I cannot help. 

Also if you could recreate this in a sample project that would help us debug as well. Thanks!

**Comment by flashmandv** [comment]
Same thing here. The patch release does not work (using from Unity). Cleaning or rebuilding makes same result: FBSDKCoreKit not found (thrown from FBSDKVideoUploader.h file)

**Comment by OutBast** [comment]
> Same thing here. The patch release does not work (using from Unity). Cleaning or rebuilding makes same result: FBSDKCoreKit not found (thrown from FBSDKVideoUploader.h file)

Same thing happening to me. Still doesn’t work for Unity.

**Comment by phonggenix** [comment]
I'm also encountering the same problem with my Unity project.

**Comment by pepejeria** [comment]
@joesus 5.11.0 does also not fix this issue.

**Comment by joesus** [comment]
@pepejeria is your project a Unity project? It looks like the current download for the Unity SDK is pegged to v5.10.0 of the iOS SDK. Will post here when that is updated.

**Comment by joesus** [comment]
This should be fixed for Unity projects with #1122

**Comment by joesus** [comment]
@pepejeria hopefully 5.11.1 patch should fix the project for Unity projects. Please post here if you are still seeing issues and we can try and figure out next steps. :D

**Comment by pepejeria** [comment]
@joesus my project is a regular iOS project. I created a new project and uploaded it here: https://github.com/pepejeria/facebooksdk

Note that the Podfile contains other pods as well. Not sure they are causing a conflict or not. 

Doing a `pod update` will add:
`${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/Assets.car`
to `Pods-FacebookSDK-resources-Debug-output-files.xcfilelist`. 

This will cause the project to stop compiling. This is a cocoapods issue though I think. Removing that line in `Pods-FacebookSDK-resources-Debug-output-files.xcfilelist` solves the issue and the project will continue to build and give the error that I mentioned above.

**Comment by joesus** [comment]
@pepejeria, thanks for uploading the sample, super helpful. I get it now. Has to do with generate_multiple_pod_projects option in the Podfile.

This is a good catch we need to be aware of this option and structure our headers accordingly so that our pods can work with this feature. The bad news is that until we get this up the workaround is to not use that flag. :/

More info here: http://blog.cocoapods.org/CocoaPods-1.7.0-beta/

**Comment by pepejeria** [comment]
@joesus alright, this used to work in 5.8, I guess it is a regression. 

Should I open a new ticket or do you handle this internally.

**Comment by joesus** [comment]
> @joesus should I open a new ticket or is this already being tracked.

Already on it! 😊

**Comment by m3rt32** [comment]
I tested with latest Unity SDK(7.18.0) and its still the same error.

**Comment by noremac** [comment]
> > @joesus should I open a new ticket or is this already being tracked.
> 
> Already on it! 😊

Hello! Is there an open issue that we could follow along on? (Or is this one it?)

**Comment by eramudeep** [comment]
getting same error.

**Comment by pepejeria** [comment]
My issue (generate_multiple_pod_projects in Pod file) is still present in 5.12.0

**Comment by weibel** [comment]
This should have been fixed in 5.12.0 by https://github.com/facebook/facebook-ios-sdk/pull/1133 but FBSDKLoginKit is still reported as v 5.11.1 by CocoaPods

**Comment by pepejeria** [comment]
My issue (generate_multiple_pod_projects in Pod file) has now been fixed. Thanks!
